### PR TITLE
Add domain config to the local install doc + remove broken slack link

### PR DESCRIPTION
**Pull request by emiliej**
Should fix issue #1106

### CODE REVIEW

File: docs/1-Installer-le-projet-en-local.md [hunk 1]
```diff
@@ -137,14 +135,29 @@ latest: Pulling from library/mariadb
...
 Cela risque de prendre un peu de temps.
 
-Une fois terminé, votre environnement de développement devrait être prêt et le projet devrait être accessible sur
-[http://127.0.0.1:8000](http://127.0.0.1:8000) (ou si vous êtes sous macOS, sur le port 8000 de la machine virtuelle de
-votre instance Docker).
+## 1.1.3 Accéder au site local
 
-## 1.1.3 Accéder à l'espace d'administration
+Une fois le projet installé, renseignez le nom de domaine choisi pour l'application locale,
+par exemple `enmarche.dev`, dans le fichier `parameters.yml` :
```
Comment (tgalopin) [inline docs/1-Installer-le-projet-en-local.md]: Could you use `app/config/parameters.yml` to be explicit?

File: docs/1-Installer-le-projet-en-local.md [hunk 2]
```diff
@@ -137,14 +135,29 @@ latest: Pulling from library/mariadb
...
+
+```
+env(ENMARCHE_DOMAIN): enmarche.dev
+```
+
+Puis, dans votre fichier hosts :
```
Comment (tgalopin) [inline docs/1-Installer-le-projet-en-local.md]: Same here: `/etc/hosts`

File: docs/1-Installer-le-projet-en-local.md (hunk 1 above)
Comment (emiliej) [inline docs/1-Installer-le-projet-en-local.md]: Sure. It's done.

**Comment by tgalopin** [comment]
Thanks @emiliej !

**Pull request merged**